tests/manage/ocs_288.py

```diff
@@ -0,0 +1,407 @@
+        CEPH_FILESYSTEM, **kwargs
+    )
+    with open(TEMP_YAML_FILE_FS, 'w') as yaml_file:
+        yaml.dump(file_y, yaml_file, default_flow_style=False)
+    log.info(f"Creating a new CephFileSystem")
+    assert CFS.create(yaml_file=TEMP_YAML_FILE_FS)
+    fs_name = file_y.get('metadata')['name']
+    fs_name = fs_name
+    POD.wait_for_resource(condition="Running", selector='app=rook-ceph-mds')
+    assert validate_cephfilesystem(fs_name=fs_name)
+    return True
+
+
+def validate_cephfilesystem(fs_name):
+    """
+    Function for validating CephFileSystem Creation
+    :param fs_name:
+    :return:
+    """
+    ceph_validate = False
+    k8s_validate = False
+    cmd = "ceph fs ls"
+    out = ocp.exec_ceph_cmd(cmd)
+    if out:
+        out = out[0]['name']
+        if out == fs_name:
+            log.info("FileSystem got created from Ceph Side")
+            ceph_validate = True
+        else:
+            log.error("FileSystem was not present at Ceph Side")
+            return False
+    result = CFS.get()
+    sample = result['items']
+    for item in sample:
+        if item.metadata.name == fs_name:
+            log.info(f"Filesystem got created from kubernetes Side")
+            k8s_validate = True
+        else:
+            log.error("Filesystem was not create at Kubernetes Side")
+            return False
+    if ceph_validate and k8s_validate:
+        return True
+    else:
+        return False
+
+
+def create_storageclass_rbd(kwargs):
+    """
+    Create a new CSI StorageClass
+    """
+    pool_name = kwargs.get('rbd_pool')
+    sc_name = kwargs.get('rbd_storageclass_name')
+    assert create_rbd_pool(pool_name=pool_name)
+    assert validate_pool_creation(pool_name)
+    if validate_pool_creation(pool_name):
+        log.info("Resource validation Passed")
+    else:
+        return 1
+    file_y = templating.generate_yaml_from_jinja2_template_with_data(
+        SC_RBD_YAML, **kwargs
+    )
+    with open(TEMP_YAML_FILE, 'w') as yaml_file:
+        yaml.dump(file_y, yaml_file, default_flow_style=False)
+    log.info(f"Creating a new RBD StorageClass")
+    assert SC.create(yaml_file=TEMP_YAML_FILE)
+    assert validate_storageclass(sc_name)
+    return True
+
+
+def get_pool_get_sc_name():
+    """
+    This will generate random pool_name and storageclass name
+    :return:
+    """
+    sc_data = {}
+    pool_name = "pool-" + ''.join(
+        [random.choice(string.ascii_lowercase) for _ in range(4)])
+    sc_name = "ocsci-csi-" + pool_name
+    sc_data['rbd_storageclass_name'] = sc_name
+    sc_data['rbd_pool'] = pool_name
+    return sc_data
+
+
+def get_storageclass():
+    """
+    Funtion for getting all storageclass
+    :return: storageclass list
+    """
+    sc_obj = SC.get()
+    sample = sc_obj['items']
+    ignore_sc = 'gp2'
+    for item in sample:
+        if item.metadata.name not in ignore_sc:
+            storageclass_list.append(item.metadata.name)
+    if len(storageclass_list) >= 1:
```

Comment:
if len(storageclass_list):

len of empty is 0 and if 0: --> False

Reply:
done